base/tuple.jl

```diff
@@ -16,8 +16,15 @@ NTuple
 
 # convenience function for extracting N from a Tuple (if defined)
 # else return `nothing` for anything else given (such as Vararg or other non-sized Union)
-_counttuple(::Type{<:NTuple{N,Any}}) where {N} = N
-_counttuple(::Type) = nothing
+
+"""
+    arity(::Type{<:Tuple})
+
+Returns the number of elements in the tuple type.
+Returns nothing for anything else.
+"""
```

Comment:
Can you please also add an `# Examples` section to this docstring? Again, see the description here for how this should look:
https://docs.julialang.org/en/v1/manual/documentation/#Documentation

Reply:
Added Examples section.